lib/node_modules/@stdlib/ndarray/base/find/benchmark/benchmark.1d_columnmajor.js

```diff
@@ -0,0 +1,143 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/* eslint-disable stdlib/no-redeclare */
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isInteger = require( '@stdlib/math/base/assert/is-integer' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var shape2strides = require( '@stdlib/ndarray/base/shape2strides' );
+var pkg = require( './../package.json' ).name;
+var find = require( './../lib' );
+
+
+// VARIABLES //
+
+var types = [ 'float64' ];
+var order = 'column-major';
+
+
+// FUNCTIONS //
+
+/**
+* Callback function.
+*
+* @param {*} value - ndarray element
+* @returns {boolean} result
+*/
+function clbk( value ) {
+	return value < 0.0;
+}
+
+/**
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} len - ndarray length
+* @param {NonNegativeIntegerArray} shape - ndarray shape
+* @param {string} xtype - ndarray data type
+* @returns {Function} benchmark function
+*/
+function createBenchmark( len, shape, xtype ) {
+	var sv;
+	var x;
+
+	x = discreteUniform( len, 1, 100 );
+	x = {
+		'dtype': xtype,
+		'data': x,
+		'shape': shape,
+		'strides': shape2strides( shape, order ),
+		'offset': 0,
+		'order': order
+	};
+	sv = {
+		'dtype': xtype,
+		'data': [ NaN ],
```

Comment:
I suggest using `array/filled`. E.g., `filledarray( NaN, 1, xtype );` <= check signature, but this should provide the idea. TL;DR: your data buffer array instance should match the `xtype`, so you need to create a single-element array containing a sentinel value.